src/backend/aspen/api/authn.py

```diff
@@ -244,32 +244,44 @@ async def get_auth_context(
     group_id: Optional[int] = Depends(get_group_context),
     user: User = Depends(get_auth_user),
     session: AsyncSession = Depends(get_db),
-    user_roles: MutableSequence[UserRole] = Depends(require_group_membership),
+    user_roles: MutableSequence[UserRole] = Depends(get_user_roles),
 ) -> AuthContext:
-    # TODO TODO TODO
-    # we need to be able to generate an authcontext without any
-    # org info *intentionally* for user self-management endpoints (get all my
-    # roles, change my username, etc) so we need to handle that case
     group = None
     roles: List[str] = []
+    groles: List[ACGroupRole] = []
```

Comment:
The changes to this method specifically allow creating authcontexts without a group context